CAM/SetGameplayCamRelativePitch.md

```diff
@@ -5,17 +5,17 @@ ns: CAM
 
 ```c
 // 0x6D0858B8EDFD2B7D 0x6381B963
-void SET_GAMEPLAY_CAM_RELATIVE_PITCH(float x, float Value2);
+void SET_GAMEPLAY_CAM_RELATIVE_PITCH(float x, float scalingFactor);
 ```
 
 ```
 Sets the camera pitch.  
 Parameters:  
 x = pitches the camera on the x axis.  
-Value2 = always seems to be hex 0x3F800000 (1.000000 float).  
+scalingFactor = always seems to be hex 0x3F800000 in native calls (1.000000 float). values of 1.0   
```

Comment:
Moved to parameters section. I'm not sure what "values of 1.0" is supposed to mean.
```suggestion
```